src/atoms/SvgIcon/index.js

```diff
@@ -17,7 +17,8 @@ class SvgIcon extends React.PureComponent {
 		this.state = { LazyIcon: () => null };
 	}
 
-	componentWillMount() {
+	/* eslint-disable camelcase */
+	UNSAFE_componentWillMount() {
```

Comment:
Would rather you changed this to use newer react, since `UNSAFE_componentWillMount` means it is deprecated and will eventually be removed.

Reply:
okay, do you mean use hooks?

Reply:
we can do even easier - remove constructor and will mount as unnecessary 
inspired by https://spectrum.chat/next-js/general/warning-expected-server-html-to-contain-a-matching-i-in-li~9ff45be2-74b7-4d08-a782-4b3210f1dfec